WordPress/metadata/release_notes.txt

```diff
@@ -1,4 +1,8 @@
-- Block editor enhancements: Added full-width and wide alignment support for Group, Cover, and Image blocks; added support for the Rounded style in the Image block; and resolved an issue where the toolbar behaved unexpectedly when adding a block.
-- Reader update: Preview images are now displayed on posts when no featured image is set.
+* [*] Block Editor: Add message that mentions are unavailable when device is offline. [https://github.com/wordpress-mobile/WordPress-Android/pull/12968]
+* [**] Block Editor: Increase tap-target of primary action on unsupported blocks. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/2608]
+* [***] Block Editor: On Jetpack connected sites, Unsupported Block Editor can be enabled via enabling Jetpack SSO setting directly from within the missing block alert. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/2610]
+* [***] Block Editor: Add support for selecting user's post when configuring the link [https://github.com/wordpress-mobile/gutenberg-mobile/pull/2484]
+* [**] Block Editor: Fix for a crash that can occur when long pressing selected images [https://github.com/wordpress-mobile/AztecEditor-Android/pull/924]
+* [***] Block Editor: Added expected double tap for focus behavior for text fields in accessibility mode [https://github.com/WordPress/gutenberg/pull/25384]
+* [*] Improved My Site layout on devices with big screens. [https://github.com/wordpress-mobile/WordPress-Android/pull/13026]
```

Comment:
Will these unformatted links with their `[*]` priority system go to GlotPress, right? Is it going to be confusing for the translators?

Reply:
Nope, they are not. In order to go to GlotPress, they have to be added to the .po file. We'll update this file with the reviewed version, before updating the .po file. 
This version is used to automatically upload them to the GH release and to PlayStore. I have manually fixed them before the release, but we should add a step to our scripts to cut everything is in square braces.